docs/configuration.rst

```diff
@@ -21,7 +21,9 @@ to the administrator.
 The repository manager provides an administration interface to the the data
 repository. It can be accessed via ``ga4gh_repo`` (or ``python repo_dev.py`` if
 developing). The ``ga4gh_repo`` commands provides a suite of commands to
-help administer a GA4GH data repository.
+help administer a GA4GH data repository. If you would like to set metadata 
+not available by the CLI, you may use the Python API directly as in 
+`scripts/prepare_compliance_data.py`.
```

Comment:
I think this asks more questions than it answers I'm afraid. We don't distribute ``prepare_compliance_data.py``, so users won't know where to find it. We can't just point users at scripts and expect them to figure it out for themselves. Having said that, documenting the admin Python API is a good idea --- people assembling a large repo should be able to code directly in Python and not have to faff about with the CLI.

But for now, I think we should  remove this change as it's likely to cause confusion.